### ISSUE TITLE
Tweakers do not appear disabled even when its value is at minimum

### ISSUE THREAD

**Issue by phet-steele** [phetsims/density#49]
1. Head into the Intro screen.
2. Adjust the volume slider by using the thumb, and drag the thumb to its min.

The mass slider will also move to its min, but you will notice the decrementing tweaker is still enabled:

![image](https://user-images.githubusercontent.com/5863899/91101028-7fc3be80-e623-11ea-8a48-7ac4aa2dab94.png)

It can be clicked, but does nothing. Two things to note:

- This happens with the volume tweaker as well, assuming the mass slider is the one that is dragged to min.
- This only appears to happen with the decrementing tweaker, not the incrementing tweaker.

Does happen in dev.14 still, but this issue is reported for dev.11.

Seen on Win 10 Chrome. For phetsims/QA/issues/507.

<details>
<summary>Troubleshooting Information</summary>

URL: https://phet-dev.colorado.edu/html/density/1.0.0-dev.11/phet/density_all_phet.html
Version: 1.0.0-dev.11 2020-06-03 18:06:51 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.125 Safari/537.36
Language: en-US
Window: 1252x923
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4095
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
</details>

**Comment by jonathanolson** [comment]
I believe this was handled in https://github.com/phetsims/density/issues/46, can you verify?

**Comment by phet-steele** [comment]
Looks great on master!

**Comment by phet-steele** [comment]
@jonathanolson I found a new procedure to get this to happen in https://phet-dev.colorado.edu/html/density/1.0.0-dev.16/phet/density_all_phet.html

1. On the intro screen, use the default Wood and slide the mass to its minimum.
2. Select Styrofoam, then switch back to Wood.

The mass value for Wood will still be at the minimum (0.4 kg), but the tweaker will look enabled.

![image](https://user-images.githubusercontent.com/5863899/123338456-1abc4c00-d506-11eb-9492-b574c70846af.png)

There is a reverse of this too by doing this step (assuming you have refreshed the sim):

1. Starting on Wood, switch to Styrofoam, then back to Wood.

The tweaker to increase mass will be disabled even though there is still plenty of room to increase the mass.

![image](https://user-images.githubusercontent.com/5863899/123338711-90c0b300-d506-11eb-86ad-48c430017891.png)

I tested all of this on Win 10 Chrome, for phetsims/QA/issues/663.
